GCM/App/login/ForgotPasswordcontroller.java

```diff
@@ -6,11 +6,11 @@
 import javafx.scene.control.TextField;
 import mainApp.GcmClient;
 
-public class ForgotPasswordController {
+public class ForgotPasswordcontroller {
```

Comment:
Again. The name changed in the master but in your end it changes back to the old name. Solve the conflict completely